tests/integration/it_utils.py

```diff
@@ -63,17 +63,19 @@
 FUNDING_AMOUNT = "1200000000"
 
 LEDGER_ACCEPT_REQUEST = GenericRequest(method="ledger_accept")
+LEDGER_ACCEPT_TIME = 1
 
 
 class AsyncTestTimer:
-    def __init__(self, timeout, client):
+    def __init__(self, timeout, client, request):
```

Comment:
```suggestion
    def __init__(self, client, timeout=LEDGER_ACCEPT_TIME,  request=LEDGER_ACCEPT_REQUEST):
```
Since that's going to be the usecase most of the time (if not all)